accord-core/src/test/java/accord/impl/basic/DelayedCommandStores.java

```diff
@@ -163,6 +166,9 @@ public DelayedCommandStore(int id, NodeTimeService time, Agent agent, DataStore
         @Override
         public void validateRead(Command current)
         {
+            if (!Invariants.testParanoia(LINEAR, LINEAR, HIGH))
```

Comment:
I would argue that for any burn test run `validateRead` should be enabled. This is a crucial part of recovery correctness, would be great if we exercise it a lot. If there are bugs here, I will be happy to assist.

Reply:
I don't think it should be enabled by default, as we don't *usually* touch this and it has a significant performance penalty. I think we should introduce a lighter-weight version that doesn't perform the intrusive diffs but still optionally loads from the journal that would also catch the majority of problems, that can be on by default.